tests/builder/help.rs

```diff
@@ -2820,3 +2820,71 @@ fn display_name_subcommand_explicit() {
         Some("child.display")
     );
 }
+
+#[test]
+fn value_terminator_in_usage_message() {
+    static EXPECTED: &str = "\
+Usage: cmd [<multiple_options>... :] [last]
+
+Arguments:
+  [<multiple_options>... :]  
+  [last]                     
+
+Options:
+  -h, --help  Print help
+";
+    let cmd = Command::new("cmd")
+        .arg(
+            Arg::new("multiple_options")
+                .num_args(0..)
+                .value_terminator(":"),
+        )
+        .arg(Arg::new("last").required(false));
+    utils::assert_output(cmd, "cmd --help", EXPECTED, false);
+}
+
+#[test]
+fn non_positional_value_terminator_in_usage_message() {
+    static EXPECTED: &str = "\
+Usage: cmd [OPTIONS] [last]
+
+Arguments:
+  [last]  
+
+Options:
+  -o, --multiple_options [<multiple_options>... :]  
+  -h, --help                                        Print help
+";
+    let cmd = Command::new("cmd")
+        .arg(
+            Arg::new("multiple_options")
+                .short('o')
+                .long("multiple_options")
+                .num_args(0..)
+                .value_terminator(":"),
+        )
+        .arg(Arg::new("last").required(false));
+    utils::assert_output(cmd, "cmd --help", EXPECTED, false);
+}
+
+#[test]
+fn required_options_with_value_terminator_in_usage_message() {
```

Comment:
Please rename this test as it is unrelated to things being required but is instead for "minimum values"